src/core/utils/assembly.ts

```diff
@@ -18,19 +18,48 @@ export interface ChromSize {
 
 /**
  * Get relative chromosome position (e.g., `100` => `{ chromosome: 'chr1', position: 100 }`)
+ * @param absPos number which is the absolute chromosome position
+ * @param assembly the assembly used to calculate which chromosome position
+ * @param returnWithinAssembly If true, then if the absolute position is before the first chromosome, it returns the
+ * first position of the first chromosome. If the absolute position is after the last chromosome, it returns the last
+ * position of the last chromosome
```

Comment:
This is the extra option I added to this function so that it doesn't return "unknown" chromosome if the axis is out of the assembly bounds.